config/frr-k8s/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- https://github.com/metallb/frr-k8s/config/default/?timeout=120&ref=main
```

Comment:
we should remember setting this to != main before merging this

Reply:
we'll have to do a tag first